rocky/rocky/scheduler.py

```diff
@@ -307,7 +307,7 @@ def post_schedule_search(self, filters: dict[str, list[dict[str, str]]]) -> Pagi
         except ConnectError:
             raise SchedulerConnectError()
 
-    def is_scheduler_ready(self, scheduler_id: str) -> bool:
+    def get_scheduler(self, scheduler_id: str) -> SchedulerResponse:
```

Comment:
Please make the trails/intervals configurable via the method arguments.
I'm pretty sure we don't want to wait 100 seconds on every page where we expect the scheduler to be available. Waiting this long is a corner case, not the default.